src/dev/flang/be/jvm/JVM.java

```diff
@@ -2230,6 +2207,89 @@ int effectId(int ecl)
     return _effectIds.add(ecl);
   }
 
+  private static String[] getDependencies()
+  {
+    String packagePath = "dev/flang/be/jvm/runtime";
+
+    Set<ClassDesc> dependencies = new HashSet<>();
+    ClassLoader classLoader = Thread.currentThread().getContextClassLoader();
+
+    try
+      {
+        Path packageDir = Paths.get(classLoader.getResource(packagePath).getPath());
```

Comment:
To create a `Path` from a `URL` via a `String` is probably not ideal, this would be the alternative, but require more try-catch blocks.

```suggestion
        Path packageDir = Path.of(classLoader.getResource(packagePath).toURI());
```